tests/specs/rpc-stacks-transaction/transaction-signing.spec.ts

```diff
@@ -53,6 +53,7 @@ test.describe('Transaction signing', () => {
     const multiSignatureTxHex = await generateMultisigUnsignedStxTransfer(
       TEST_ACCOUNT_2_STX_ADDRESS,
       amount,
+      100,
```

Comment:
This test was failing bc the high fee warning popup was being triggered w/out a fee provided.